.devcontainer/scripts/.zshrc_devcontainer

```diff
@@ -5,6 +5,17 @@ repo_root="/workspaces/slang"
 # Hermit
 eval "$("${repo_root}/bin/hermit" shell-hooks --print --zsh)"
 
+if [[ ! -f "${repo_root}/target/shell-completions/_infra" ]]; then
```

Comment:
nit: I suggest adding `zsh` to that file name (since we will support other shells):

```suggestion
if [[ ! -f "${repo_root}/target/shell-completions/zsh_infra" ]]; then
```

Reply:
Only ZSH uses that convention (https://github.com/NomicFoundation/slang/pull/739#discussion_r1484361131) but changed to always include the shell suffix in https://github.com/NomicFoundation/slang/pull/739/commits/a189f3900d53326ff3e27bbdb149210287063747